cmd/sync-tags/main.go

```diff
@@ -73,6 +73,7 @@ func main() {
 	skipFetch := flag.Bool("skip-fetch", false, "skip fetching tags")
 	generateGodeps := flag.Bool("generate-godeps", false, "regenerate Godeps.json from go.mod")
 	mappingOutputFile := flag.String("mapping-output-file", "", "a file name to write the source->dest hash mapping to ({{.Tag}} is substituted with the tag name, {{.Branch}} with the local branch name)")
+	basePackage := flag.String("base-pkg", "k8s.io", "the canonical package name (eg. k8s.io)")
```

Comment:
FYI this is a "breaking change" for non-k8s repos, but they'll need this if they decide to use go 1.13 anyway.

We also default `base_package` to k8s.io in `util.sh` today, unless specified.

Reply:
I don't object to requiring the base package, but I wouldn't expect this to default to k8s.io. I'd expect invokers to be required to provide it now

Reply:
I thought we have this concept already in some form. Target org or so?

Reply:
> I thought we have this concept already in some form. Target org or so?


We don't have any info about the base package name in sync-tags, we "derive" it from the package path:

https://github.com/kubernetes/publishing-bot/blob/ec388bdeb34233252eb18babc440d82a385c91b8/cmd/sync-tags/gomod.go#L40

I updated to use the path to avoid adding another flag. PTAL.